src/scripts.js

```diff
@@ -525,8 +526,22 @@ async function determinePantryDisplayEventTarget(event) {
   populatePantryDisplay();
 }
 
+async function addCustomIngredientToPantry() {
+  let customIngredient = pantryDropdown.value;
+  let customAmount = document.querySelector('#addIngredientAmount')
+  console.log(customIngredient)
+  console.log(customAmount.value)
+  await addIngredientToPantry(customIngredient, customAmount.value)
+  populatePantryDisplay()
+  customAmount.value = ""
+  
+  
+}
+
 //event listeners
 
+customAddIngredientButton.addEventListener('click', addCustomIngredientToPantry)
```

Comment:
We just want to make sure to error handle the things we don't want them to be putting in the input field!